### DISCUSSION TITLE
Geolocation Tracker with XIAO ESP32S3 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#340] [Q&A] [answered]
# Geolocation Tracker with XIAO ESP32S3 | Seeed Studio Wiki

Making an ultra-compact global positioning tracker with the XIAO ESP32S3.

https://wiki.seeedstudio.com/geolocation_tracker_with_XIAO_ESP32S3/#comments

**Comment by rmwerder** [comment]
the STP files are just for a housing for the MC. Having those who are for the whole tracker would be nice ;-)

**Reply by MatthewJeffson** [reply]
Hi! I will check if we got this. Thank you!

**Reply by rmwerder** [reply, marked as answer]
well, that's embarassing. For me! I'm truely impressed with the efforts you people provide with this website, and then I'm makeing a nonsense request :(
OFC I made a typo. Writing "stp" instead of "stl". But what I wanted was not the small rectangle housing for the ESP32S3 but the round one for the display - which can be found at https://wiki.seeedstudio.com/get_start_round_display/
*facepalm* and sorry, sorry, sorry for wasting your time
Rainer

**Reply by MatthewJeffson** [reply]
That's really ok, Rainer. :D 
No problem at all.
We had something urgent to do last week, and I'm sorry I didn't get back to you until now
